package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
-    "watch": "NODE_ENV=development webpack --watch",
+    "watch": "set NODE_ENV=development && webpack --watch --progress",
```

Comment:
Is this intentional? In my tests, the `set` command does not work properly on Unix and does not actually set the environment variable properly. What's wrong with `NODE_ENV=development webpack --watch --progress`?

Reply:
I was getting the error `'NODE_ENV' is not recognized as an internal or external command`. After a quick research, I found [this StackOverflow question](https://stackoverflow.com/questions/11928013/node-env-is-not-recognized-as-an-internal-or-external-command-operable-comman).

It seems that on Windows you need to write `set` and `&&` for it to work properly 😕.

However, I think there are some packages that solve this cross-platform inconvenience, I will take a look at that 🙂, for the time being, I don't mind discarding these lines whenever I commit something.

Reply:
Adding this module to my environment fixed the issue:
`npm install -g win-node-env`